src/proposals/20240926/SparkEthereum_20240926.t.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/SparkTestBase.sol';
+
+interface ITolled {
+    function kiss(address) external;
+}
+
+contract SparkEthereum_20240926Test is SparkEthereumTestBase {
+
+    address internal constant CBBTC            = 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf;
+    address internal constant CBBTC_PRICE_FEED = 0xb9ED698c9569c5abea716D1E64c089610a3768B6;
```

Comment:
I think a test validating the price from the feed would be great

Reply:
added